manifests/install.pp

```diff
@@ -1,18 +1,23 @@
+# Class networkmanager::install
 class networkmanager::install {
-  package { 'network-manager':
-    ensure => $::networkmanager::version,
-  }
-  case $::networkmanager::gui {
-    'gnome': {
-      package { 'network-manager-gnome':
-        ensure => $::networkmanager::version,
-      }
+  include ::networkmanager
```

Comment:
`networkmanager::install` is a "private" class that's supposed to be called from the `networkmanager` class. There is no point in including the latter here.